tests/activemq5-unit-tests/pom.xml

```diff
@@ -460,6 +460,8 @@
                   <exclude>**/org/apache/activemq/ProducerFlowControlTest.java</exclude>
                   <!-- exclude tests that are on client side only -->
                   <exclude>**/org/apache/activemq/transport/tcp/TransportConnectorInvalidSocketOptionsTest.java</exclude>
+                  <!-- exclude tests for features not implemented in Artemis -->
+                  <exclude>**/org/apache/activemq/MessageEvictionTest.java</exclude>
```

Comment:
if that is how it is done regarding non-applicable activemq5 tests in Artemis, then sure, I can just delete the test.